.travis.yml

```diff
@@ -77,13 +77,14 @@ before_script:
         brew services start kafka
         sleep 10
     else
-        sudo docker-compose -f scripts/kafka-docker-compose.yml up > kafka_zookeeper.log &
+        #sudo docker-compose -f scripts/kafka-docker-compose.yml up > kafka_zookeeper.log &
```

Comment:
Can you leave a comment here explaining the differences between `kafka-docker-compose` and `bitnami-...`? Or, if you are sure the `bitnami` one is the one for us, might as well delete this line and that file.

Reply:
I will remove the old line now that I am confident the bitnami container works.